package.json

```diff
@@ -1,42 +1,48 @@
 {
-  "name": "nemo-screenshot",
```

Comment:
It may be _this_ module that is out of compliance, per the stated JSHint config:
https://github.com/paypal/nemo-docs/blob/master/contributing.md#general-formatting